src/foremast/gcp_iam/service_account.py

```diff
@@ -0,0 +1,64 @@
+#   Foremast - Pipeline Tooling
+#
+#   Copyright 2020 Redbox Automated Retail, LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""GCP Service Account helpers"""
+
+import logging
+
+import googleapiclient.discovery
+
+LOG = logging.getLogger(__name__)
+
+
+def create_service_account(credentials, project_id, name):
```

Comment:
This is creating the unique service account for mapping app to account right?

Foobar app == foobar service account

Reply:
Yes, the service account is made in the service account project defined in the GCP Environment in Foremcast config.  The service account name is the app name.  So say the project is coolapp/greatteam you'd end up with:
coolappgreatteam@prod-project-name.iam.gserviceaccount.com, coolappgreatteam@stage-project-name.iam.gserviceaccount.com, etc.